.storybook/recipes/ProjectCard/ProjectCard.module.css

```diff
@@ -0,0 +1,57 @@
+@import '../../../src/design-tokens/mixins.css';
+
+/*------------------------------------*\
+    #PROJECTCARD
+\*------------------------------------*/
+
+/**
+ * 1) 
+ */
+.project-card {
+  position: relative;
+  transition: all var(--eds-anim-fade-quick) var(--eds-anim-fade-quick);
+}
+
+.project-card__drag-handle {
```

Comment:
Is this styling used anywhere? I can't find the class name in JS

Reply:
Good catch! Originally had it on the `ProjectCard` but the item itself handles that now.